vllm/v1/engine/__init__.py

```diff
@@ -42,10 +43,17 @@ class EngineCoreOutput(
 
     request_id: str
     new_token_ids: List[int]
-    finished: bool
+    new_logprobs: List[torch.Tensor] = []
+    new_logprobs_token_ids: List[torch.Tensor] = []
```

Comment:
Its odd to have a list of torch tensors. I think this should be `List[float]` and `List[int]`